plugins/woocommerce-admin/client/two-column-tasks/completed-header.tsx

```diff
@@ -97,10 +97,20 @@ export const TaskListCompletedHeader: React.FC<
 		}
 	}, [ hasSubmittedScore ] );
 
-	const submitScore = ( recordedScore: number, comments?: string ) => {
+	const submitScore = ( {
+		firstScore,
+		secondScore,
+		comments,
+	}: {
+		firstScore: number;
+		secondScore?: number;
+		comments?: string;
+	} ) => {
 		recordEvent( 'ces_feedback', {
 			action: CUSTOMER_EFFORT_SCORE_ACTION,
-			score: recordedScore,
+			score: firstScore,
+			score_second_question: secondScore ? secondScore : null,
+			score_combined: score + ( secondScore ? secondScore : 0 ),
```

Comment:
I think that there is a problem here with the `score + ( secondScore ? secondScore : 0 )`, if we use `firstScore` instead of `score` we will be sure that `score_combined` never will be `NaN`. Otherwise, for example, after adding a positive review (like 😍) to the task list experience (after finishing it) `score_combined` will be `NaN`.

![Screenshot 2022-11-23 at 15 38 30](https://user-images.githubusercontent.com/1314156/203627472-29d780ee-3c64-4477-ba9e-d6f9dfd66c1d.png)

I would also simplify this logic with something like:
```
score_second_question: secondScore ?? null,
score_combined: firstScore + ( secondScore ?? 0 ),
```

Reply:
Ah! That `score` was a total typo; nice catch @octaedro  👍🏻 . This is done in 07da5547863e1a00b208626e4fd95d31ad40f63a.